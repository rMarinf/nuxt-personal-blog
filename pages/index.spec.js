import { shallowMount } from '@vue/test-utils'
import Blog from './index.vue'

test('Blog', () => {
  // render the component
  const wrapper = shallowMount(Blog)

  expect(wrapper.element).toMatchSnapshot()
})
