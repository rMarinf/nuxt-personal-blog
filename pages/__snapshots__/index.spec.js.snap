// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog 1`] = `
<div
  id="blog"
>
  
  BLOG

</div>
`;
